infra/blueprint-test/README.md

```diff
@@ -0,0 +1,409 @@
+# 1. Background and Overview
+
+Integration testing has been instrumental in establishing a complete and functional CI pipeline. Similar to most quality software engineering modules and applications, GCP Blueprints employ an integration testing framework that is triggered as part of the CI process as well.
+
+Apart from the necessity of including a testing framework as part of our GCP blueprints we took additional steps to ensure the following:
+
+- Easy to learn and to setup as part of the blueprints development process
+- Compatible with both Terraform and KRM based GCP blueprints
+
+Considering the above, our test framework has been developed (details in the following sections) with backward compatibility to allow for current tests to keep functioning.
+
+```
+Note: If you have a question about the test framework, feel free to ask it on our user group. 
+Feature requests can also be submitted as Issues.
+```
+
+# 2. Framework Concepts
+
+```
+Note: The best reference documentation for the framework is the autogenerated documentation.
+```
+
+The test framework uses Golang with the testing package at its core to allow for creation and setup of GCP blueprints integration tests.
+
+```
+Note: The test runners for the framework are published on GitHub and can be downloaded in your local workspace using `go init <package_name> & go get -t` for test development.
```

Comment:
Not sure what you meant by this

Reply:
Actually this is not needed now. Initially I documented to explain how to setup the go module paths but we already have them setup or plan to have them setup for all repos so this is redundant. I'll take it out.